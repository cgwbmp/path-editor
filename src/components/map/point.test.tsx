import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Map from '.';
import MapPoint from './point';

describe('<MapPoint />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render((
      <Map center={[0, 0]}>
        <MapPoint id="1" title="Title" place={[0, 0]} />
      </Map>
    ), div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('DOM structure not changed', () => {
    const component = renderer.create((
      <Map center={[0, 0]}>
        <MapPoint id="1" title="Title" place={[0, 0]} />
      </Map>
    ));
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
