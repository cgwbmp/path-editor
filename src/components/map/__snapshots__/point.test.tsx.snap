// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapPoint /> DOM structure not changed 1`] = `null`;
